webapp/channels/src/components/file_preview_modal/file_preview_modal_info/file_preview_modal_info.tsx

```diff
@@ -79,6 +79,7 @@ const FilePreviewModalInfo: React.FC<Props> = (props: Props) => {
                     username={user?.username}
                     url={imageURLForUser(props.post.user_id, user?.last_picture_update)}
                     className='file-preview-modal__avatar'
+                    alt=''
```

Comment:
The profile picture is decorative here, so we set its alt text to empty to prevent it from being read out by a11y tools